starknet_py/net/account/account_client.py

```diff
@@ -73,6 +75,34 @@ async def _get_nonce(self) -> int:
         )
         return nonce
 
+    async def get_balance(self, token_address: AddressRepresentation = None) -> int:
+        """
+        Checks account's balance of specified token._
+
+        :param token_address: Address of the ERC20 contract.
+                              If not specified it will be mainnet or testnet payment token address.
+        :return: Account's balance of token.
+        """
+        if token_address is None:
```

Comment:
Maybe we should consider making token_address a property of AccountClient?

I'm not sure if there's use case where someone would want to use multiple token addresses per account but in that case we could also allow for passing a different `token_address` as a parameter to `get_balance` that would override address set on AccountClient.